documentation/flow_chart.md

```diff
@@ -0,0 +1,29 @@
+## Design Flowchart
+- Below we can see a diagram that explains how the flask app will interact with the backend. All of the run check calls are performed on the AWS side per the lambda functions. The App only talks to the dynamo ddb via creating monitors or grabbing log information that is store via the lambda functions.
+```mermaid
+flowchart TD
+
+subgraph UserSide[User Side]
+    UI[Web UI] -->|REST calls| API[Flask API]
```

Comment:
You may want to update this slightly.

The REST API is called directly to create/update monitors by a developer.

The UI is used strictly to read data and do other UI-necessary tasks like create projects and (un)pause monitors. It won't necessarily utilize REST API conventions.